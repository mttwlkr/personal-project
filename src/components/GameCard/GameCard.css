.game-container-card {
  width: 400px;
  height: 350px;
  background: #fff;
  margin: 2em 1em 0em 1em;
}

.game-card-image {
  width: 100%;
}

.game-card-info {
  display: flex;
  flex-direction: column;
  padding: 1% 4%;
}

.game-card-title {
  margin-top: 0em;
  margin-bottom: 0;
  padding: 0.2em 0;
  font-weight: 400;
  font-size: 16px;
  line-height: 1.2em;
}

.game-card-loc-date {
  font-size: 12px;
  font-weight: 400;
  font-style: italic;
  opacity: 0.6;
}

.game-card-type {
  margin: 0em;
  color: white;
  padding: 3px 10px;
  margin: .3em auto;
  text-transform: uppercase;
}

.slvsh-style,
.film-style,
.instabanger-style {
  font-weight: 800;
}

.slvsh-style {
  background: #53e2ff;
}

.film-style {
  background: black;
}

.instabanger-style {
  background: red;
}
