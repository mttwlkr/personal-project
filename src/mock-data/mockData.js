export const mockAPIRidersData = [
  {
    "id": 30,
    "name": "Nicky Keefer",
    "sponsor": "Atomic",
    "avatar": "http://slvsh_prod.s3.amazonaws.com/riders/avatars/000/000/030/original/keefer.jpg?1419916201",
    "country": "us",
    "birthdate": "1991-02-16",
    "slvsh_avg": 167,
    "url": "http://www.slvsh.com/riders/30.json",
    "age": 27
  },
  {
    "id": 23,
    "name": "Torin Yater-Wallace",
    "sponsor": "Red Bull",
    "avatar": "http://slvsh_prod.s3.amazonaws.com/riders/avatars/000/000/023/original/torin.jpg?1417500198",
    "country": "us",
    "birthdate": "1995-12-01",
    "slvsh_avg": 444,
    "url": "http://www.slvsh.com/riders/23.json",
    "age": 22
  },
  {
    "id": 19,
    "name": "Lupe Hagearty",
    "sponsor": "Nordica",
    "avatar": "http://slvsh_prod.s3.amazonaws.com/riders/avatars/000/000/019/original/lupe.jpg?1416289155",
    "country": "us",
    "birthdate": "1992-08-13",
    "slvsh_avg": 444,
    "url": "http://www.slvsh.com/riders/19.json",
    "age": 25
  }
]

export const mockAPIGamesData = {
"current_page": 1,
"total_pages": 42,
  "posts": [
    {
      "id": 869,
      "created_at": "2018-04-05T15:16:50.775Z",
      "updated_at": "2018-04-05T15:16:50.775Z",
      "video_host": "youtube",
      "vimeo_id": "dJQmXDZW1ig",
      "type": "Film",
      "location": "Grandvalira, Andorra",
      "video_url": "https://www.youtube.com/embed/dJQmXDZW1ig",
      "title": "Recap Edit || Slvsh Cup&nbsp;Grandvalira",
      "poster_img": "http://slvsh_prod.s3.amazonaws.com/films/poster_imgs/000/000/869/medium/RecapCover16x9.jpg?1522941405",
      "post_class": "film",
      "publish_date": "1 day",
      "href": "/films/869",
      "description": "Thats it folks!! As the 16th and Final game of the cup has made its way online we have to close out this years Slvsh Cup. We can't thank you, the fans, enough for all the views and comments on this years cup. This has been by far our best event to date and we can only hope to do it bigger and better next time! As a bonus and thank you, we give you the #SlvshCupGrandvalira Recap edit. This has to be one of the funnest weeks for all the athletes no matter how they end up in the contest, and this video should prove that! A huge shout out to Grandvalira, Monster Energy and Volkl Skis for their continued support of Slvsh and making this event happen!! Film/edit by Andrew Napier Music PhantomOnDaTrack",
      "formatted_publish_date": "04.05.2018",
      "tags": [
        { "name": "Joss Christensen" },
        { "name": "Alex Bellemare" },
        { "name": "Steve Stepp" },
        { "name": "Johan Berg" },
        { "name": "charlie lasser" },
        { "name": "slvsh cup" },
        { "name": "alex hall" },
        { "name": "jake carney" },
        { "name": "Andrew Napier" },
        { "name": "emil granbom" },
        { "name": "hugo burvall" },
        { "name": "edit" },
        { "name": "slvsh cup grandvalira" },
        { "name": "Quinn Wolferman" },
        { "name": "henrik harlaut" },
        { "name": "Ferdinand Dahl" },
        { "name": "Simone Canal" },
        { "name": "Hunter Hess" },
        { "name": "Ethan Swadburg" },
        { "name": "Tom Ritsch" },
        { "name": "James Woods" },
        { "name": "elias syrja" },
        { "name": "Aleksi Patja" }
      ],
      "tags_string": "Joss Christensen, Alex Bellemare, Steve Stepp, Johan Berg, charlie lasser, slvsh cup, alex hall, jake carney, Andrew Napier, emil granbom, hugo burvall, edit, slvsh cup grandvalira, Quinn Wolferman, henrik harlaut, Ferdinand Dahl, Simone Canal, Hunter Hess, Ethan Swadburg, Tom Ritsch, James Woods, elias syrja, Aleksi Patja"
    },
    {
      "id": 836,
      "created_at": "2018-04-03T13:05:25.506Z",
      "updated_at": "2018-04-03T13:05:25.506Z",
      "video_host": "youtube",
      "vimeo_id": "b_9xjpVljtM",
      "type": "Game",
      "location": "Grandvalira, AD",
      "video_url": "https://www.youtube.com/embed/b_9xjpVljtM",
      "title": "Final || Henrik Harlaut vs. Quinn Wolferman || Slvsh Cup&nbsp;Grandvalira",
      "poster_img": "http://slvsh_prod.s3.amazonaws.com/games/poster_imgs/000/000/836/medium/HenrikvsQuinnCover16x9.jpg?1522760710",
      "post_class": "slvsh",
      "publish_date": "3 days",
      "href": "/games/836",
      "winner": {
        "id": 150,
        "name": "Henrik Harlaut",
        "created_at": "2017-10-10T12:48:54.345Z",
        "updated_at": "2018-03-15T13:23:05.198Z",
        "sponsor": "Monster Energy",
        "avatar_file_name": "henrik600.jpg",
        "avatar_content_type": "image/jpeg",
        "avatar_file_size": 203220,
        "avatar_updated_at": "2017-10-10T12:48:53.810Z",
        "country": "se",
        "birthdate": "1995-01-01",
        "slvsh_avg": 500
      },
      "loser": {
        "id": 110,
        "name": "Quinn Wolferman",
        "created_at": "2016-08-02T07:30:32.665Z",
        "updated_at": "2016-08-02T07:32:11.888Z",
        "sponsor": "Tall Tee",
        "avatar_file_name": "QuinnWolferman.jpg",
        "avatar_content_type": "image/jpeg",
        "avatar_file_size": 51763,
        "avatar_updated_at": "2016-08-02T07:30:32.264Z",
        "country": "us",
        "birthdate": "1992-10-06",
        "slvsh_avg": 500
      },
      "game_type": "S.L.V.S.H.",
      "winner_id": 150,
      "loser_id": 110,
      "winner_avatar=": "http://slvsh_prod.s3.amazonaws.com/riders/avatars/000/000/150/thumb/henrik600.jpg?1507639733",
      "loser_avatar=": "http://slvsh_prod.s3.amazonaws.com/riders/avatars/000/000/110/thumb/QuinnWolferman.jpg?1470123032",
      "sorted_riders": [
        { 
          "name": "Henrik Harlaut",
          "avatar_url": "http://slvsh_prod.s3.amazonaws.com/riders/avatars/000/000/150/thumb/henrik600.jpg?1507639733"
        },
        {
        "name": "Quinn Wolferman",
        "avatar_url": "http://slvsh_prod.s3.amazonaws.com/riders/avatars/000/000/110/thumb/QuinnWolferman.jpg?1470123032"
        }
      ],
      "winner_score": 5,
      "loser_score": 5,
      "description": "Grandvalira, Monster Energy, and Volkl Skis are proud to present Slvsh Cup Grandvalira 2018! The final match up and the battle for $10,000 comes down to Henrik Harlaut and Quinn Wolferman. Could we ask for a better match up? Don't think so!",
      "formatted_publish_date": "04.03.2018",
      "tags": [
        { "name": "slvsh cup" },
        { "name": "game" },
        { "name": "Quinn Wolferman" },
        { "name": "henrik harlaut" }
      ],
      "tags_string": "slvsh cup, game, Quinn Wolferman, henrik harlaut"
    },
    {
      "id": 835,
      "created_at": "2018-04-02T14:10:08.000Z",
      "updated_at": "2018-04-02T14:10:08.000Z",
      "video_host": "youtube",
      "vimeo_id": "_ygdXqvSstg",
      "type": "Game",
      "location": "Grandvalira, AD",
      "video_url": "https://www.youtube.com/embed/_ygdXqvSstg",
      "title": "Consolation || Alex Hall vs. Hunter Hess || Slvsh Cup&nbsp;Grandvalira",
      "poster_img": "http://slvsh_prod.s3.amazonaws.com/games/poster_imgs/000/000/835/medium/AlexVHunterCover16x9.jpg?1522678205",
      "post_class": "slvsh",
      "publish_date": "4 days",
      "href": "/games/835",
      "winner": {
        "id": 80,
        "name": "Alex Hall",
        "created_at": "2015-12-08T14:30:14.125Z",
        "updated_at": "2016-08-09T08:52:12.365Z",
        "sponsor": "Shred",
        "avatar_file_name": "Alexhall.jpg",
        "avatar_content_type": "image/jpeg",
        "avatar_file_size": 89077,
        "avatar_updated_at": "2015-12-08T14:30:12.918Z",
        "country": "us",
        "birthdate": "1998-10-21",
        "slvsh_avg": 500
      },
      "loser": {
        "id": 153,
        "name": "Hunter Hess",
        "created_at": "2017-11-14T13:14:45.116Z",
        "updated_at": "2017-11-14T13:20:26.628Z",
        "sponsor": "Volkl",
        "avatar_file_name": "482179_614260521934935_889184546_n.jpg",
        "avatar_content_type": "image/jpeg",
        "avatar_file_size": 266086,
        "avatar_updated_at": "2017-11-14T13:14:44.512Z",
        "country": "us",
        "birthdate": "1997-10-01",
        "slvsh_avg": 500
      },
      "game_type": "S.L.V.S.H.",
      "winner_id": 80,
      "loser_id": 153,
      "winner_avatar=": "http://slvsh_prod.s3.amazonaws.com/riders/avatars/000/000/080/thumb/Alexhall.jpg?1449585012",
      "loser_avatar=": "http://slvsh_prod.s3.amazonaws.com/riders/avatars/000/000/153/thumb/482179_614260521934935_889184546_n.jpg?1510665284",
      "sorted_riders": [
        {
          "name": "Alex Hall",
          "avatar_url": "http://slvsh_prod.s3.amazonaws.com/riders/avatars/000/000/080/thumb/Alexhall.jpg?1449585012"
        },
        {
          "name": "Hunter Hess",
          "avatar_url": "http://slvsh_prod.s3.amazonaws.com/riders/avatars/000/000/153/thumb/482179_614260521934935_889184546_n.jpg?1510665284"
        }
      ],
      "winner_score": 5,
      "loser_score": 5,
      "description": "Grandvalira, Monster Energy, and Volkl Skis are proud to present Slvsh Cup Grandvalira 2018! Alex Hall and Hunter Hess battle it out for the 3rd place title in the consolation round. Watch to see who walks away with 3rd place and $2,500.00",
      "formatted_publish_date": "04.02.2018",
      "tags": [
        { "name": "slvsh cup" },
        { "name": "alex hall" },
        { "name": "monster energy" },
        { "name": "sunset park" },
        { "name": "game" },
        { "name": "grandvalira" },
        { "name": "Hunter Hess" }
      ],
      "tags_string": "slvsh cup, alex hall, monster energy, sunset park, game, grandvalira, Hunter Hess"
    }
  ]
} 