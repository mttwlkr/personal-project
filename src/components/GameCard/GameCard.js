import React from 'react';
import './GameCard.css';

const GameCard = ({game}) => {
  return (
    <div className='game-container-card'>
      <h3>{game.title}</h3>
      <img className='game-card-thumbnail' src={game.poster_img}/>
      <p>{game.type}</p>
      <p>{`${game.publish_date} ago`}</p>
      <p>{game.location}</p>
    </div>
  )
}


export default GameCard;

// { id
//       created_at
//       updated_at
//       video_host
//       vimeo_id
//       type
//       location
//       video_url
//       title
//       poster_img
//       post_class
//       publish_date
//       href
//       description
//       formatted_publish_date
//       tags
//       tags_string }