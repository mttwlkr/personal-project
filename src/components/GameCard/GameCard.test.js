import React from 'react';
import { shallow } from 'enzyme';
import GameCard from './GameCard.js';
import { mockAPIGamesData } from '../../mock-data/mockData';

describe('GameCard', () => {

  let wrapper;
  let mockGame = mockAPIGamesData.posts[1]
  let mockHandleRoute = jest.fn()

  beforeEach(() => {
    wrapper = shallow(<GameCard
      handleRoute={mockHandleRoute}
      game={mockGame} // game_postclass is a game
    />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })

  it('should add styling based on game_postclass', () => {
    let mockGame = mockAPIGamesData.posts[0] // game_postclass is a film
    wrapper = shallow(<GameCard 
      game={mockGame}
      handleRoute={mockHandleRoute}
    />)
    expect(wrapper).toMatchSnapshot()
  })

  it('should invoke handleRoute based on click', () => {
    wrapper.find('.game-container-card').simulate('click')
    expect(mockHandleRoute).toHaveBeenCalledWith(836)
  })
})