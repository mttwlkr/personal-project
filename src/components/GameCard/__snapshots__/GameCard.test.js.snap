// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameCard should add styling based on game_postclass 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <GameCard
    game={
        Object {
            "created_at": "2018-04-05T15:16:50.775Z",
            "description": "Thats it folks!! As the 16th and Final game of the cup has made its way online we have to close out this years Slvsh Cup. We can't thank you, the fans, enough for all the views and comments on this years cup. This has been by far our best event to date and we can only hope to do it bigger and better next time! As a bonus and thank you, we give you the #SlvshCupGrandvalira Recap edit. This has to be one of the funnest weeks for all the athletes no matter how they end up in the contest, and this video should prove that! A huge shout out to Grandvalira, Monster Energy and Volkl Skis for their continued support of Slvsh and making this event happen!! Film/edit by Andrew Napier Music PhantomOnDaTrack",
            "formatted_publish_date": "04.05.2018",
            "href": "/films/869",
            "id": 869,
            "location": "Grandvalira, Andorra",
            "post_class": "film",
            "poster_img": "http://slvsh_prod.s3.amazonaws.com/films/poster_imgs/000/000/869/medium/RecapCover16x9.jpg?1522941405",
            "publish_date": "1 day",
            "tags": Array [
              Object {
                "name": "Joss Christensen",
              },
              Object {
                "name": "Alex Bellemare",
              },
              Object {
                "name": "Steve Stepp",
              },
              Object {
                "name": "Johan Berg",
              },
              Object {
                "name": "charlie lasser",
              },
              Object {
                "name": "slvsh cup",
              },
              Object {
                "name": "alex hall",
              },
              Object {
                "name": "jake carney",
              },
              Object {
                "name": "Andrew Napier",
              },
              Object {
                "name": "emil granbom",
              },
              Object {
                "name": "hugo burvall",
              },
              Object {
                "name": "edit",
              },
              Object {
                "name": "slvsh cup grandvalira",
              },
              Object {
                "name": "Quinn Wolferman",
              },
              Object {
                "name": "henrik harlaut",
              },
              Object {
                "name": "Ferdinand Dahl",
              },
              Object {
                "name": "Simone Canal",
              },
              Object {
                "name": "Hunter Hess",
              },
              Object {
                "name": "Ethan Swadburg",
              },
              Object {
                "name": "Tom Ritsch",
              },
              Object {
                "name": "James Woods",
              },
              Object {
                "name": "elias syrja",
              },
              Object {
                "name": "Aleksi Patja",
              },
            ],
            "tags_string": "Joss Christensen, Alex Bellemare, Steve Stepp, Johan Berg, charlie lasser, slvsh cup, alex hall, jake carney, Andrew Napier, emil granbom, hugo burvall, edit, slvsh cup grandvalira, Quinn Wolferman, henrik harlaut, Ferdinand Dahl, Simone Canal, Hunter Hess, Ethan Swadburg, Tom Ritsch, James Woods, elias syrja, Aleksi Patja",
            "title": "Recap Edit || Slvsh Cup&nbsp;Grandvalira",
            "type": "Film",
            "updated_at": "2018-04-05T15:16:50.775Z",
            "video_host": "youtube",
            "video_url": "https://www.youtube.com/embed/dJQmXDZW1ig",
            "vimeo_id": "dJQmXDZW1ig",
          }
    }
    handleRoute={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          alt="slvsh-video-thumbnail"
          className="game-card-image"
          src="http://slvsh_prod.s3.amazonaws.com/films/poster_imgs/000/000/869/medium/RecapCover16x9.jpg?1522941405"
/>,
        <div
          className="game-card-info"
>
          <p
                    className="game-card-type film-style"
          >
                    THEATER
          </p>
          <h3
                    className="game-card-title"
          >
                    Recap Edit || Slvsh Cup Grandvalira
          </h3>
          <p
                    className="game-card-loc-date"
          >
                    Grandvalira, Andorra, 1 day ago
          </p>
</div>,
      ],
      "className": "game-container-card",
      "onClick": [Function],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "slvsh-video-thumbnail",
          "className": "game-card-image",
          "src": "http://slvsh_prod.s3.amazonaws.com/films/poster_imgs/000/000/869/medium/RecapCover16x9.jpg?1522941405",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p
              className="game-card-type film-style"
>
              THEATER
</p>,
            <h3
              className="game-card-title"
>
              Recap Edit || Slvsh Cup Grandvalira
</h3>,
            <p
              className="game-card-loc-date"
>
              Grandvalira, Andorra, 1 day ago
</p>,
          ],
          "className": "game-card-info",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "THEATER",
              "className": "game-card-type film-style",
            },
            "ref": null,
            "rendered": "THEATER",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Recap Edit || Slvsh Cup Grandvalira",
              "className": "game-card-title",
            },
            "ref": null,
            "rendered": "Recap Edit || Slvsh Cup Grandvalira",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Grandvalira, Andorra, 1 day ago",
              "className": "game-card-loc-date",
            },
            "ref": null,
            "rendered": "Grandvalira, Andorra, 1 day ago",
            "type": "p",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="slvsh-video-thumbnail"
            className="game-card-image"
            src="http://slvsh_prod.s3.amazonaws.com/films/poster_imgs/000/000/869/medium/RecapCover16x9.jpg?1522941405"
/>,
          <div
            className="game-card-info"
>
            <p
                        className="game-card-type film-style"
            >
                        THEATER
            </p>
            <h3
                        className="game-card-title"
            >
                        Recap Edit || Slvsh Cup Grandvalira
            </h3>
            <p
                        className="game-card-loc-date"
            >
                        Grandvalira, Andorra, 1 day ago
            </p>
</div>,
        ],
        "className": "game-container-card",
        "onClick": [Function],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "slvsh-video-thumbnail",
            "className": "game-card-image",
            "src": "http://slvsh_prod.s3.amazonaws.com/films/poster_imgs/000/000/869/medium/RecapCover16x9.jpg?1522941405",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p
                className="game-card-type film-style"
>
                THEATER
</p>,
              <h3
                className="game-card-title"
>
                Recap Edit || Slvsh Cup Grandvalira
</h3>,
              <p
                className="game-card-loc-date"
>
                Grandvalira, Andorra, 1 day ago
</p>,
            ],
            "className": "game-card-info",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "THEATER",
                "className": "game-card-type film-style",
              },
              "ref": null,
              "rendered": "THEATER",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Recap Edit || Slvsh Cup Grandvalira",
                "className": "game-card-title",
              },
              "ref": null,
              "rendered": "Recap Edit || Slvsh Cup Grandvalira",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Grandvalira, Andorra, 1 day ago",
                "className": "game-card-loc-date",
              },
              "ref": null,
              "rendered": "Grandvalira, Andorra, 1 day ago",
              "type": "p",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`GameCard should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <GameCard
    game={
        Object {
            "created_at": "2018-04-03T13:05:25.506Z",
            "description": "Grandvalira, Monster Energy, and Volkl Skis are proud to present Slvsh Cup Grandvalira 2018! The final match up and the battle for $10,000 comes down to Henrik Harlaut and Quinn Wolferman. Could we ask for a better match up? Don't think so!",
            "formatted_publish_date": "04.03.2018",
            "game_type": "S.L.V.S.H.",
            "href": "/games/836",
            "id": 836,
            "location": "Grandvalira, AD",
            "loser": Object {
              "avatar_content_type": "image/jpeg",
              "avatar_file_name": "QuinnWolferman.jpg",
              "avatar_file_size": 51763,
              "avatar_updated_at": "2016-08-02T07:30:32.264Z",
              "birthdate": "1992-10-06",
              "country": "us",
              "created_at": "2016-08-02T07:30:32.665Z",
              "id": 110,
              "name": "Quinn Wolferman",
              "slvsh_avg": 500,
              "sponsor": "Tall Tee",
              "updated_at": "2016-08-02T07:32:11.888Z",
            },
            "loser_avatar=": "http://slvsh_prod.s3.amazonaws.com/riders/avatars/000/000/110/thumb/QuinnWolferman.jpg?1470123032",
            "loser_id": 110,
            "loser_score": 5,
            "post_class": "slvsh",
            "poster_img": "http://slvsh_prod.s3.amazonaws.com/games/poster_imgs/000/000/836/medium/HenrikvsQuinnCover16x9.jpg?1522760710",
            "publish_date": "3 days",
            "sorted_riders": Array [
              Object {
                "avatar_url": "http://slvsh_prod.s3.amazonaws.com/riders/avatars/000/000/150/thumb/henrik600.jpg?1507639733",
                "name": "Henrik Harlaut",
              },
              Object {
                "avatar_url": "http://slvsh_prod.s3.amazonaws.com/riders/avatars/000/000/110/thumb/QuinnWolferman.jpg?1470123032",
                "name": "Quinn Wolferman",
              },
            ],
            "tags": Array [
              Object {
                "name": "slvsh cup",
              },
              Object {
                "name": "game",
              },
              Object {
                "name": "Quinn Wolferman",
              },
              Object {
                "name": "henrik harlaut",
              },
            ],
            "tags_string": "slvsh cup, game, Quinn Wolferman, henrik harlaut",
            "title": "Final || Henrik Harlaut vs. Quinn Wolferman || Slvsh Cup&nbsp;Grandvalira",
            "type": "Game",
            "updated_at": "2018-04-03T13:05:25.506Z",
            "video_host": "youtube",
            "video_url": "https://www.youtube.com/embed/b_9xjpVljtM",
            "vimeo_id": "b_9xjpVljtM",
            "winner": Object {
              "avatar_content_type": "image/jpeg",
              "avatar_file_name": "henrik600.jpg",
              "avatar_file_size": 203220,
              "avatar_updated_at": "2017-10-10T12:48:53.810Z",
              "birthdate": "1995-01-01",
              "country": "se",
              "created_at": "2017-10-10T12:48:54.345Z",
              "id": 150,
              "name": "Henrik Harlaut",
              "slvsh_avg": 500,
              "sponsor": "Monster Energy",
              "updated_at": "2018-03-15T13:23:05.198Z",
            },
            "winner_avatar=": "http://slvsh_prod.s3.amazonaws.com/riders/avatars/000/000/150/thumb/henrik600.jpg?1507639733",
            "winner_id": 150,
            "winner_score": 5,
          }
    }
    handleRoute={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          alt="slvsh-video-thumbnail"
          className="game-card-image"
          src="http://slvsh_prod.s3.amazonaws.com/games/poster_imgs/000/000/836/medium/HenrikvsQuinnCover16x9.jpg?1522760710"
/>,
        <div
          className="game-card-info"
>
          <p
                    className="game-card-type slvsh-style"
          >
                    S.L.V.S.H.
          </p>
          <h3
                    className="game-card-title"
          >
                    Final || Henrik Harlaut vs. Quinn Wolferman || Slvsh Cup Grandvalira
          </h3>
          <p
                    className="game-card-loc-date"
          >
                    Grandvalira, AD, 3 days ago
          </p>
</div>,
      ],
      "className": "game-container-card",
      "onClick": [Function],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "slvsh-video-thumbnail",
          "className": "game-card-image",
          "src": "http://slvsh_prod.s3.amazonaws.com/games/poster_imgs/000/000/836/medium/HenrikvsQuinnCover16x9.jpg?1522760710",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p
              className="game-card-type slvsh-style"
>
              S.L.V.S.H.
</p>,
            <h3
              className="game-card-title"
>
              Final || Henrik Harlaut vs. Quinn Wolferman || Slvsh Cup Grandvalira
</h3>,
            <p
              className="game-card-loc-date"
>
              Grandvalira, AD, 3 days ago
</p>,
          ],
          "className": "game-card-info",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "S.L.V.S.H.",
              "className": "game-card-type slvsh-style",
            },
            "ref": null,
            "rendered": "S.L.V.S.H.",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Final || Henrik Harlaut vs. Quinn Wolferman || Slvsh Cup Grandvalira",
              "className": "game-card-title",
            },
            "ref": null,
            "rendered": "Final || Henrik Harlaut vs. Quinn Wolferman || Slvsh Cup Grandvalira",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Grandvalira, AD, 3 days ago",
              "className": "game-card-loc-date",
            },
            "ref": null,
            "rendered": "Grandvalira, AD, 3 days ago",
            "type": "p",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="slvsh-video-thumbnail"
            className="game-card-image"
            src="http://slvsh_prod.s3.amazonaws.com/games/poster_imgs/000/000/836/medium/HenrikvsQuinnCover16x9.jpg?1522760710"
/>,
          <div
            className="game-card-info"
>
            <p
                        className="game-card-type slvsh-style"
            >
                        S.L.V.S.H.
            </p>
            <h3
                        className="game-card-title"
            >
                        Final || Henrik Harlaut vs. Quinn Wolferman || Slvsh Cup Grandvalira
            </h3>
            <p
                        className="game-card-loc-date"
            >
                        Grandvalira, AD, 3 days ago
            </p>
</div>,
        ],
        "className": "game-container-card",
        "onClick": [Function],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "slvsh-video-thumbnail",
            "className": "game-card-image",
            "src": "http://slvsh_prod.s3.amazonaws.com/games/poster_imgs/000/000/836/medium/HenrikvsQuinnCover16x9.jpg?1522760710",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p
                className="game-card-type slvsh-style"
>
                S.L.V.S.H.
</p>,
              <h3
                className="game-card-title"
>
                Final || Henrik Harlaut vs. Quinn Wolferman || Slvsh Cup Grandvalira
</h3>,
              <p
                className="game-card-loc-date"
>
                Grandvalira, AD, 3 days ago
</p>,
            ],
            "className": "game-card-info",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "S.L.V.S.H.",
                "className": "game-card-type slvsh-style",
              },
              "ref": null,
              "rendered": "S.L.V.S.H.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Final || Henrik Harlaut vs. Quinn Wolferman || Slvsh Cup Grandvalira",
                "className": "game-card-title",
              },
              "ref": null,
              "rendered": "Final || Henrik Harlaut vs. Quinn Wolferman || Slvsh Cup Grandvalira",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Grandvalira, AD, 3 days ago",
                "className": "game-card-loc-date",
              },
              "ref": null,
              "rendered": "Grandvalira, AD, 3 days ago",
              "type": "p",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
